templates/zerver/help/configure_email_notifications.md

```diff
@@ -0,0 +1,15 @@
+# Configure email notifications
+
+Zulip allows you to configure your email notification preferences. You have a choice to receive or not receive email notification when you're offline.
+
+You can easily change your email notification settings by following these steps.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner of the right sidebar.
+2. Select **Settings** from the dropdown menu that appears.
+
+    ![Settings](/static/images/help/settings.png)
+
+3. Scroll down to the **Notifications** section (after the **Display Settings** section) to view the message display options.
+4. Under the **Other notifications I want** section of **Notifications** you should find a check box with the lable **Digest emails when I'm away**. When this checkbox is selected you will receive email notifications whenever you're offline.
+    ![Settings](/static/images/help/notifications-settings.png)
```

Comment:
I can't find this image in `/static/images/help`. If it is your own image, you forgot to add it to the PR.